1_fetch/in/target_sciencebase_attributes.csv

```diff
@@ -1,7 +1,32 @@
-IMPERVIOUS_COVER_2011,IMPV11,NLCD 2011 percent imperviousness ,percent,https://www.sciencebase.gov/catalog/item/57057a9be4b0d4e2b7571fbb
-RIP_TREE_CANOPY_2011,CNPY11_BUFF100,NLCD 2011 percent tree canopy in 100 meter riparian buffer ,percent,https://www.sciencebase.gov/catalog/item/570572e2e4b0d4e2b75718bc
+CLIMATE_PET,PET,"PRISM 30-year average Potential Evapotranspiration, 1971-2000 per NHDPlus version 2 catchment",millimeters per year,https://www.sciencebase.gov/catalog/item/56f96ed1e4b0a6037df06a2d
+CLIMATE_PRECIP,PPT7100_ANN,"Mean annual precip (mm) for the watershed, from 800m PRISM data.  30 years period of record 1971-2000. ",mm/year,https://www.sciencebase.gov/catalog/item/573b70a7e4b0dae0d5e3ae85
+HYDROLOGY_RUNOFF,RUN7100,Estimated 30-year (1971-2000) average annual runoff in millimeters per year,millimeters per year,https://www.sciencebase.gov/catalog/item/578f8ad8e4b0ad6235cf6e43
+HYDROLOGY_BFI,BFI,"Base Flow Index (BFI), The BFI is a ratio of base flow to total streamflow, expressed as a percentage and ranging from 0 to 100. Base flow is the sustained, slowly varying component of streamflow, usually attributed to ground-water discharge to a stream.",percent,https://www.sciencebase.gov/catalog/item/5669a8e3e4b08895842a1d4f
+HYDROLOGY_WTDEPTH,EWT,Average depth to water table relative to the land surface(meters),meters,https://www.sciencebase.gov/catalog/item/56f97456e4b0a6037df06b50
+HYDROLOGY_NPDES,NPDES_MAJ,Number of major NPDES sites ,count,https://www.sciencebase.gov/catalog/item/57c9d89ce4b0f2f0cec192da
+HYDROLOGY_DAMS,NDAMSYYYY,"Number of dams built on or before YYYY per NHDPlus version 2 catchment, where YYYY is the last year for the decade of record (for example 1960 spans 1951 - 1960).",count,https://www.sciencebase.gov/catalog/item/58c301f2e4b0f37a93ed915a
+HYDROLOGY_DAMS,NID_STORAGEYYYY,"The maximum dam storage (in acre-feet) defined as the total storage space in all reservoirs in a flowline catchment below the maximum attainable water surface elevation, including any surcharge storage, of dams built on or before YYYY. Value is based on dams built on or before YYYY, where YYYY is the last year for the decade of record (for example 1960 spans 1951 - 1960). The exceptions are 1930 and 2013 (1930 and before and 2010 to 2010 respectively).",acre-feet,https://www.sciencebase.gov/catalog/item/58c301f2e4b0f37a93ed915a
+HYDROLOGY_DAMS,NORM_STORAGEYYYY,"The normal dam storage (in acre-feet) defined as the total storage space in all reservoirs in a flowline catchment that is below the normal retention level, including dead and inactive storage and excluding any flood control or surcharge storage. Value is based on dams built on or before YYYY, where YYYY is the last year for the decade of record (for example 1960 spans 1951 - 1960). The exceptions are 1930 and 2013 (1930 and before and 2010 to 2010 respectively).",acre-feet,https://www.sciencebase.gov/catalog/item/58c301f2e4b0f37a93ed915a
```

Comment:
`The only other attribute that I had thought of is a calculation of the amount of reservoir storage/mean flow at each site. I have seen this as a way to capture the degree of hydrologic alteration. `

Thanks, Galen - great suggestion! After our conversation last week I added these two variables from the larger Wieczorek dataset (`NID_STORAGE` and `NORM_STORAGE`). Are these the variables you were referring to? If not, do you have a link to alternative datasets that address reservoir storage?

Reply:
Yeah I think that sounds great!